cli/command/service.go

```diff
@@ -10,6 +18,260 @@ func NewServiceCommand(b *CommandBase) *ServiceCommand {
+				Action:    s.scale,
+				ArgsUsage: "NAME COUNT",
+			},
+			{
+				Name:      "update",
+				Usage:     "run a new dploy on a service",
+				Action:    s.update,
+				ArgsUsage: "NAME DEPLOY",
+			},
+		},
+	}
+}
+
+func (s *ServiceCommand) create(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "ENVIRONMENT", "NAME", "DEPLOY")
+	if err != nil {
+		return err
+	}
+
+	deployID, err := s.resolveSingleEntityIDHelper("deploy", args["DEPLOY"])
+	if err != nil {
+		return err
+	}
+
+	environmentID, err := s.resolveSingleEntityIDHelper("environment", args["ENVIRONMENT"])
+	if err != nil {
+		return err
+	}
+
+	var loadBalancerID string
+	if loadBalancerName := c.String("loadbalancer"); loadBalancerName != "" {
+		id, err := s.resolveSingleEntityIDHelper("load_balancer", loadBalancerName)
+		if err != nil {
+			return err
+		}
+
+		loadBalancerID = id
+	}
+
+	req := models.CreateServiceRequest{
+		DeployID:       deployID,
+		EnvironmentID:  environmentID,
+		LoadBalancerID: loadBalancerID,
+		ServiceName:    args["NAME"],
+	}
+
+	jobID, err := s.client.CreateService(req)
+	if err != nil {
+		return err
+	}
+
+	onCompleteFN := func(serviceID string) error {
+		service, err := client.WaitForDeployment(s.client, serviceID, c.GlobalDuration(config.FLAG_TIMEOUT))
+		if err != nil {
+			return err
+		}
+
+		return s.printer.PrintServices(service)
+	}
+
+	return s.waitOnJobHelper(c, jobID, "creating", onCompleteFN)
+}
+
+func (s *ServiceCommand) delete(c *cli.Context) error {
+	deleteFN := func(serviceID string) (string, error) {
+		return s.client.DeleteService(serviceID)
+	}
+
+	return s.deleteHelper(c, "service", deleteFN)
+}
+
+func (s *ServiceCommand) list(c *cli.Context) error {
+	serviceSummaries, err := s.client.ListServices()
+	if err != nil {
+		return err
+	}
+
+	return s.printer.PrintServiceSummaries(serviceSummaries...)
+}
+
+func (s *ServiceCommand) logs(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "NAME")
+	if err != nil {
+		return err
+	}
+
+	serviceID, err := s.resolveSingleEntityIDHelper("service", args["NAME"])
+	if err != nil {
+		return err
+	}
+
+	query := buildLogQueryHelper(serviceID, c.String("start"), c.String("end"), c.Int("tail"))
+
+	logs, err := s.client.ReadServiceLogs(serviceID, query)
+	if err != nil {
+		return err
+	}
+
+	return s.printer.PrintLogs(logs...)
+}
+
+func (s *ServiceCommand) read(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "NAME")
+	if err != nil {
+		return err
+	}
+
+	serviceIDs, err := s.resolver.Resolve("service", args["NAME"])
+	if err != nil {
+		return err
+	}
+
+	services := make([]*models.Service, len(serviceIDs))
+	for i, serviceID := range serviceIDs {
+		service, err := s.client.ReadService(serviceID)
+		if err != nil {
+			return err
+		}
+
+		services[i] = service
+	}
+
+	return s.printer.PrintServices(services...)
+}
+
+func (s *ServiceCommand) scale(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "NAME", "COUNT")
```

Comment:
If you make the changes above, `NAME` must become `SERVICE_NAME`

Reply:
✅